bot/across_the_stars_cmds.py

```diff
@@ -0,0 +1,33 @@
+from bot.across_the_stars.planets import Planets
+import discord
+from discord.ext import commands
+import os
+
+
+class AcrossTheStarsCmds(commands.Cog):
+
+    def __init__(self, client):
+        self.client = client
+        self.planets = Planets()
+
+    @commands.command(aliases=['planets'])
+    async def planetas(self, ctx, *, args):
```

Comment:
Como o bot utiliza os nomes dos parâmetros das funções de comando para documentação, é legal utilizarmos um nome mais descritivo. Também é importante explicarmos o que o comando faz.
```suggestion
    async def planetas(self, ctx, *, region):
    """
    Lista todos os planetas disponíveis da região fornecida
    """
```